hubspot3/tickets.py

```diff
@@ -15,6 +18,15 @@ class TicketsClient(BaseClient):
     :see: https://developers.hubspot.com/docs/methods/tickets/tickets-overview
     """
 
+    __default_batch_properties = ["subject"]
+
+    class Recency:
```

Comment:
If this was an enum, I'd recommend using the standard enum module https://docs.python.org/3/library/enum.html.

However, this does not look like an enum but rather a constant placeholder, because the string values are used. I recommend changing your documentation to reflect that -- in enums you do not use the value other than for equality comparison.

Reply:
Just following the rest of the code's standard